docs/public/specifications/tests/Address-TP/address02.md

```diff
@@ -1,46 +1,102 @@
-## ADDRESS02: Reverse DNS entry exists for name server IP address
+# ADDRESS02: Reverse DNS entry exists for name server IP address
 
-### Test case identifier
-**ADDRESS02** Reverse DNS entry should exist for name server IP address
+## Test case identifier
+**ADDRESS02** 
 
-### Objective
+## Table of contents
 
-Some anti-spam techniques use reverse DNS lookup to allow incoming traffic.
-In order to prevent name servers to be blocked or blacklisted, DNS 
-administrators should publish PTR records associated to name server
-addresses.
+* [Objective](#objective)
+* [Scope](#scope)
+* [Inputs](#inputs)
+* [Summary](#summary)
+* [Test procedure](#test-procedure)
+* [Outcome(s)](#outcomes)
+* [Special procedural requirements](#special-procedural-requirements)
+* [Intercase dependencies](#intercase-dependencies)
 
-TODO: Technical reference to be found
+## Objective
 
-### Inputs
+Best curent practices dictates that imternet reachable hosts should have a
+reverse DNS entry, as various services on the Internet, for instance spam 
+filters, may consider this when determining the trustworthiness of the host.
+Se [RFC1912] section 2.1 and [RFC1033] page 11 for additional information.
+
+## Scope
+
+This test checks for the existence of PTR records for the corresponding reverse
+domains of the name servers IP address.
```

Comment:
This test checks for the existence of PTR records for the corresponding reverse
domains of the name servers IP address.